src/components/FooterNavbar.jsx

```diff
@@ -0,0 +1,137 @@
+            fill=""
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M22.5 3C11.7321 3 3 10.9527 3 20.7643C3 28.6847 6.4125 47.4825 40.0032 66.4849C40.6049 66.8218 41.2957 67 42 67C42.7043 67 43.3951 66.8218 43.9968 66.4849C77.5875 47.4825 81 28.6847 81 20.7643C81 10.9527 72.2679 3 61.5 3C50.7321 3 42 13.7662 42 13.7662C42 13.7662 33.2679 3 22.5 3Z"
+              stroke=""
+              strokeWidth="3"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            />
+          </svg>
+        </NavLink>
+        <NavLink to="/co2-rechner">
+          <svg
+            className="icon"
+            width="40"
+            height="40"
+            viewBox="0 0 61 58"
+            fill=""
+            xmlns="http://www.w3.org/2000/svg"
+          >
+            <path
+              d="M10.614 19.9375H15.86V24.9255C15.86 25.056 15.9973 25.1575 16.165 25.1575H19.825C19.9928 25.1575 20.13 25.056 20.13 24.9183V19.9375H25.376C25.5132 19.9375 25.62 19.807 25.62 19.6475V16.1675C25.62 16.008 25.5133 15.8775 25.3684 15.8775H20.13V10.8895C20.13 10.759 19.9928 10.6575 19.825 10.6575H16.165C15.9973 10.6575 15.86 10.759 15.86 10.8895V15.8775H10.614C10.4768 15.8775 10.37 16.008 10.37 16.1675V19.6475C10.37 19.807 10.4768 19.9375 10.614 19.9375ZM35.624 19.9375H50.386C50.5233 19.9375 50.63 19.807 50.63 19.6475V16.1675C50.63 16.008 50.5233 15.8775 50.3784 15.8775H35.624C35.4868 15.8775 35.38 16.008 35.38 16.1675V19.6475C35.38 19.807 35.4868 19.9375 35.624 19.9375ZM35.624 39.15H50.386C50.5233 39.15 50.63 39.0195 50.63 38.86V35.38C50.63 35.2205 50.5233 35.09 50.3784 35.09H35.624C35.4868 35.09 35.38 35.2205 35.38 35.38V38.86C35.38 39.0195 35.4868 39.15 35.624 39.15ZM35.624 46.69H50.386C50.5233 46.69 50.63 46.5595 50.63 46.4V42.92C50.63 42.7605 50.5233 42.63 50.3784 42.63H35.624C35.4868 42.63 35.38 42.7605 35.38 42.92V46.4C35.38 46.5595 35.4868 46.69 35.624 46.69ZM20.7019 40.8175L25.3684 35.3872C25.6963 35.0102 25.4217 34.4375 24.9185 34.4375H21.045C20.8696 34.4375 20.7019 34.51 20.5951 34.6478L18.0026 37.6637L15.4101 34.6478C15.3556 34.5839 15.2871 34.532 15.2095 34.4957C15.1318 34.4594 15.0468 34.4395 14.9603 34.4375H11.0791C10.5759 34.4375 10.3014 35.0102 10.6292 35.3872L15.2958 40.8175L10.5149 46.3855C10.1794 46.7625 10.4539 47.3353 10.9571 47.3353H14.8306C15.006 47.3353 15.1737 47.2627 15.2805 47.125L17.9874 43.9712L20.6943 47.125C20.8086 47.2555 20.9764 47.3353 21.1441 47.3353H25.0176C25.5209 47.3353 25.7954 46.7625 25.4675 46.3855L20.7019 40.8175V40.8175ZM58.56 0H2.44C1.09037 0 0 1.03675 0 2.32V55.68C0 56.9632 1.09037 58 2.44 58H58.56C59.9096 58 61 56.9632 61 55.68V2.32C61 1.03675 59.9096 0 58.56 0ZM55.815 53.07H5.185V4.93H55.815V53.07Z"
+              fill=""
+              stroke=""
+              strokeWidth="2"
+              strokeLinecap="round"
+              strokeLinejoin="round"
+            />
+          </svg>
+        </NavLink>
+      </NavbarStyled>
+      <Outlet />
```

Comment:
Is the Outlet really needed here?

Reply:
No it's actually not needed so I deleted it. Please review my code again - I'm looking forward to your approval :)